core/services/ocr2/plugins/ocr2keeper/evm21/core/trigger.go

```diff
@@ -0,0 +1,103 @@
+package core
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/automation_utils_2_1"
+	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
```

Comment:
I think this needs something like a `goimports -w -local github.com/smartcontractkit/chainlink core/services/ocr2/plugins/ocr2keeper/evm21/core/trigger.go`?

Reply:
done